tests/opensource-suites/collections/screenshots/collectionsPage.test.ts

```diff
@@ -0,0 +1,22 @@
+import {Page} from '@playwright/test';
+
+import {CollectionsPageQa} from '../../../../src/shared';
+import {openTestPage, slct} from '../../../utils';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+
+datalensTest.describe('Collections page', () => {
+    datalensTest(
+        'Collections page displays list of collections and workbooks @screenshot',
+        async ({page}: {page: Page}) => {
+            await page.clock.setFixedTime(new Date('2025-01-01T10:00:00'));
```

Comment:
Since the collection contains an entity that is used in other tests and is being changed (e2e workbook), a fixed time will not help much in this case. I think it would be cleaner to create a separate collection with a new (possibly empty) workbook

Reply:
I'm fixing time here to make stable this part of screenshot
"last modified" there depends on current date

<img width="154" alt="Screenshot 2024-11-28 at 15 02 56" src="https://github.com/user-attachments/assets/ae903703-e713-43d9-bf3c-c68a1c8f17b8">